codes/javascript/chapter_hashing/hash_map_open_addressing.js

```diff
@@ -14,149 +14,157 @@ class Pair {
 
 /* 开放寻址哈希表 */
 class HashMapOpenAddressing {
-    #size; // 键值对数量
-    #capacity; // 哈希表容量
-    #loadThres; // 触发扩容的负载因子阈值
-    #extendRatio; // 扩容倍数
-    #buckets; // 桶数组
-    #removed; // 删除标记
-
     /* 构造方法 */
     constructor() {
-        this.#size = 0;
-        this.#capacity = 4;
-        this.#loadThres = 2.0 / 3.0;
-        this.#extendRatio = 2;
-        this.#buckets = new Array(this.#capacity).fill(null);
-        this.#removed = new Pair(-1, '-1');
+        this.size = 0; // 键值对数量
+        this.capacity = 4; // 哈希表容量
+        this.loadThres = 2.0 / 3.0; // 触发扩容的负载因子阈值
+        this.extendRatio = 2; // 扩容倍数
+        this.buckets = Array(this.capacity).fill(null); // 桶数组
+        this.TOMBSTONE = new Pair(-1, '-1'); // 删除标记
```

Comment:
These are all private fields, so we need to add # before them. When we use private fields, we need to define them before using them in the constructor, please reset them, thanks!